website/docs/r/loadbalancer_nat_pool.html.markdown

```diff
@@ -8,7 +8,7 @@ description: |-
 
 # azurerm_lb_nat_pool
 
-Manages a Load Balancer NAT pool.
+Manages a Load Balancer NAT pool. This resource is intended to use with virtual machine scale sets.
```

Comment:
This can be used with VMs as well so can we revert this?
```suggestion
Manages a Load Balancer NAT pool.
```

Reply:
Thanks for reviewing.
I might be wrong, but I've made those note(s) because in _ARM Template reference_ (**API version 2019-04-01**) and _REST API reference_ (**API version 2019-07-01**) regarding creating\updating Load Balancer resource in Azure there are some clarifications about using\referencing NAT Pools\NAT Rules:
https://docs.microsoft.com/en-us/azure/templates/microsoft.network/2019-04-01/loadbalancers#LoadBalancerPropertiesFormat
https://docs.microsoft.com/en-us/rest/api/load-balancer/loadbalancers/createorupdate#request-body

Reply:
Talking internally it sounds like while this cannot be used with a VM, it can be used with other azure resource. So instead could we perhaps add a note instead?
```suggestion
Manages a Load Balancer NAT pool. 

-> **NOTE:** This resource cannot be used with with virtual machines, instead use the `azurerm_lb_nat_rule` resource.
```